Use string-pixel-width to get text width

jhen0409 opened the pull request:
The original text width calculation seems wrong to set `undefined`, the text width will be `0` for some case.

jhen0409 commented:
Currently it will crop words for some case, so I'm not sure if it is better. Maybe we could use lib like [`string-pixel-width`](https://github.com/adambisek/string-pixel-width) but need support more fonts.

orta commented:
I think string pixel length is probably a better call 👍

orta commented:
I'm ok with it's general subset of fonts, allowing any of those but falling back to times new roman or something

jhen0409 commented:
I changed to use `string-pixel-width`, added `font-family`, `font-size` for render text, and set `margin: 0` for body (default: 8). It looks better now. :)

orta commented:
Yep - looks fab, let's get this out.
